lib/kinesisClient.js

```diff
@@ -7,22 +7,32 @@ const kinesis = new AWS.Kinesis({region : config.KINESIS_REGION});
 
 /**
  * Writes to Kinesis
- * @param {Array} uuids -
+ * @param {Array/String} uuids - takes an Array of uuids or a single uuid
  * @param {String} event -
  * @param {Array} eventData -
  * @returns {Promise} result -
  */
 function writeToKinesis (uuids, event, eventData) {
   const operation = 'kinesisClient.writeToKinesis';
-  logger.info({operation, uuids, event});
+  let uuidsArr = [];
```

Comment:
don't necessarily have to assign to an empty array here

Reply:
Sorry don't understand what is wrong with that. Please can you explain.